sketch_map_tool/models.py

```diff
@@ -34,6 +34,20 @@ class Layer(StrEnum):
     ESRI_WORLD_IMAGERY = "esri-world-imagery"
 
 
+class MapAttributions:
+    def __init__(self):
+        self.attributions = {
+            "osm": "Powered by OpenStreetMap<br />©openstreetmap.org/copyright",
+            "esri-world-imagery": "Powered by Esri<br />© Esri, Maxar, GeoEye, "
+            "Earthstar Geographics, CNES/Airbus DS, USD,"
+            " USGS, AeroGRID, IGN, and the GIS User Community",
+        }
+
+    def get_attribution(self, key):
+        if key in self.attributions:
+            return self.attributions[key]
```

Comment:
I suggest using dataclass. It's more concise and its the same as in the other "models" in this module. Something like this:
```
@dataclass(frozen=True)
class Attribution:
    osm = "Powered by OpenStreetMap<br />©openstreetmap.org/copyright"
    esri-world-imagery = "..."
```